scripts/dev.ts

```diff
@@ -836,6 +841,71 @@ async function main() {
         description: 'Proposal with value > 1',
       },
     },
+    // Approve tokens to be locked from acc[1]
+    {
+      type: 'approve',
+      from: accounts[1],
+      data: {
+        asset: 'SWPR',
+        address: 'SWPRGuild-vault',
+        amount: MAX_UINT,
+      },
+    },
+    // Lock tokens
+    {
+      type: 'guild-lockTokens',
+      from: accounts[1],
+      data: {
+        guildName: 'SWPRGuild',
+        amount: web3.utils.toWei('30').toString(),
+      },
+    },
+    // After locking tokens acc[1] creates a proposal to allow guild to make SWPR transfers
+    {
+      type: 'guild-createProposal',
+      from: accounts[1],
+      data: {
+        guildName: 'SWPRGuild',
+        to: ['PermissionRegistry'],
+        callData: [
+          new web3.eth.Contract(PermissionRegistry.abi).methods
+            .setETHPermission(
+              networkContracts.addresses.SWPRGuild,
+              networkContracts.addresses.SWPR,
+              ERC20_TRANSFER_SIGNATURE,
+              web3.utils.toWei('0').toString(),

```

Comment:
I think this shouldn't be a string

Reply:
It's confusing. Some fn calls are using string.
Removed!

Reply:
Hmm, perhaps it is handled fine here